### PR TITLE
Use `main` as entry point for application if file isn't passed as argument

### LINKED ISSUE #47: CLI should take entry point file from package.json#main if not provided as argument
That would make my `npm start` script slightly more micro.

### PR DESCRIPTION
**Pull request by julianduque**
It will look for `main` property in package.json and will try to use it as main file when argument isn't provided.

Fixes #47

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,7 +64,6 @@
     "gulp-ext": "1.0.0",
     "gulp-task-listing": "1.0.1",
     "request-promise": "2.0.0",
-    "resumer": "0.0.0",
```
Comment (rauchg) [inline package.json:67]: I thought we were using this in tests?

**Comment by julianduque** [comment]
@rauchg you added it here on dependencies: https://github.com/zeit/micro/commit/01171e1291e4bc59c6873652c430e6b4f9e641e1, so it's duplicated, so it should be on deps or devDeps?

**Comment by rauchg** [comment]
`devDeps`, my bad

**Comment by rauchg** [comment]
Awesome contributions @julianduque

**Pull request merged**